ver/us/splat.yaml

```diff
@@ -322,7 +324,8 @@ segments:
     - [0x5A610, .data, effects]
     - [0x5B320, .data, 5B320]
     - [0x6B360, .data, world/world]
-    - [0x6EAC0, data] # has scripts
+    - [0x6EAC0, .data, 38F00]
+    - [0x6EF80, data, 6EAC0]
```

Comment:
Could you rename this .data.s file to 6EF80 just to match the new start address?